tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -423,6 +423,22 @@ describe(__filename, () => {
     expect(link).toHaveProp('href', `/addon/${addon.slug}/versions/beta`);
   });
 
+  it('does not link to beta versions if they are disabled', () => {
```

Comment:
This test should move to https://github.com/mozilla/addons-frontend/blob/master/tests/unit/core/reducers/test_addons.js